frontend/public/data/rbd/wca_admbnda_adm2_ocha.json

```diff
@@ -791,7 +791,7 @@
14 ], [ -4.518927141, 6.188627001 ], [ -4.518689702, 6.191010546 ], [ -4.518359806, 6.192956329 ], [ -4.518017333, 6.194947928 ], [ -4.517567587, 6.197246735 ], [ -4.517230984, 6.199514135 ], [ -4.516911098, 6.201505671 ], [ -4.5165721, 6.202738768 ], [ -4.515966943, 6.204312311 ], [ -4.515151417, 6.206283224 ], [ -4.514555096, 6.208579627 ], [ -4.51412717, 6.210316572 ], [ -4.513520298, 6.212578529 ], [ -4.51283219, 6.21437952 ], [ -4.512012988, 6.215959892 ], [ -4.510980121, 6.217478705 ], [ -4.509821351, 6.2197072 ], [ -4.508546238, 6.221542651 ], [ -4.50733668, 6.223494906 ], [ -4.506614624, 6.22528356 ], [ -4.506015324, 6.227143643 ], [ -4.505280532, 6.229023894 ], [ -4.504843773, 6.230646129 ], [ -4.503202767, 6.235104575 ], [ -4.502971262, 6.236018598 ], [ -4.502373157, 6.237212873 ], [ -4.501714698, 6.238635095 ], [ -4.501152981, 6.240312513 ], [ -4.500382264, 6.241090163 ], [ -4.499633682, 6.242105136 ], [ -4.498970337, 6.243003997 ], [ -4.497507563, 6.245214703 ], [ -4.496254237, 6.247096994 ], [ -4.49504999, 6.24876211 ], [ -4.493448192, 6.250557219 ], [ -4.492194123, 6.251887875 ], [ -4.490976091, 6.253081646 ], [ -4.489734493, 6.254332567 ], [ -4.488319345, 6.255832106 ], [ -4.487206865, 6.257407083 ], [ -4.486217899, 6.259019016 ], [ -4.485488638, 6.259992666 ], [ -4.483982838, 6.261502626 ], [ -4.482498406, 6.26308148 ], [ -4.4816101, 6.264143746 ], [ -4.480785618, 6.264805667 ], [ -4.479378055, 6.26530702 ], [ -4.477704807, 6.266135808 ], [ -4.476208573, 6.266543578 ], [ -4.474468583, 6.266740079 ], [ -4.472740384, 6.26691337 ], [ -4.470659596, 6.267241379 ], [ -4.469400498, 6.267619077 ], [ -4.468415009, 6.267876183 ], [ -4.467732722, 6.267587012 ], [ -4.466939993, 6.266584859 ], [ -4.466326519, 6.265654547 ], [ -4.465830561, 6.265036467 ], [ -4.464820592, 6.263605358 ], [ -4.463962845, 6.26247451 ], [ -4.463324786, 6.261635827 ], [ -4.462811749, 6.260764575 ], [ -4.462312957, 6.259733331 ], [ -4.461602241, 6.257951512 ], [ -4.46117784, 6.256554368 ], [ -4.460911886, 6.255159605 ], [ -4.460595199, 6.254005201 ], [ -4.459232966, 6.252728222 ], [ -4.457603043, 6.251870238 ], [ -4.45559599, 6.251889366 ], [ -4.453299143, 6.252304371 ], [ -4.451467526, 6.253165165 ], [ -4.449836791, 6.253547795 ], [ -4.448413037, 6.253715633 ], [ -4.446999575, 6.253929789 ], [ -4.445811155, 6.254744708 ], [ -4.444398388, 6.255521806 ], [ -4.442901666, 6.255963961 ], [ -4.441277088, 6.256024378 ], [ -4.439440594, 6.255944069 ], [ -4.438009071, 6.255342081 ], [ -4.43672705, 6.254571246 ], [ -4.435704589, 6.25380535 ], [ -4.434538653, 6.253450188 ], [ -4.433474547, 6.253671266 ], [ -4.43222133, 6.254347858 ], [ -4.430780286, 6.255433608 ], [ -4.429170499, 6.256470643 ], [ -4.427789102, 6.257374226 ], [ -4.426146448, 6.258376864 ], [ -4.425361446, 6.258839649 ], [ -4.424618456, 6.259277449 ], [ -4.423086972, 6.260350433 ], [ -4.421922625, 6.261109285 ], [ -4.421020724, 6.261688817 ], [ -4.420264905, 6.262294517 ], [ -4.418398219, 6.263387614 ] ] ] ] } },
```

Comment:
@laurentS what changed here?

Reply:
@wadhwamatic ^

Reply:
Thanks for catching and fixing this @laurentS. I see that the error is in the original source boundary file. I'll let the team in Dakar know. The fix you made is correct.

I see others that should be fixed too: CI1302, CI1303, CI1304

Reply:
Indeed, and we need to update the links for the ones we change in the admin data files.

Reply:
@wadhwamatic I fixed the others. However for 01 and 02 we have a problem because there are two data info with the same keys in the admin data level datasets and we don't have a way to know which datapoint is for which region. In https://github.com/WFP-VAM/prism-app/pull/974/commits/d231af6af4e716f6a0d5553b1380440dac26c22b I was able to fix the ica and multi risk datasets for 03 and 04 but not for 01 and 02